graspologic/plot/plot.py

```diff
@@ -671,6 +676,226 @@ def pairplot(
     return pairs
 
 
+def _plot_ellipse_and_data(
+    data, X, j, k, means, covariances, ax, label_palette, cluster_palette, alpha
+):
+    r"""
+    plot_ellipse makes a scatter plot from the two dimensions j,k where j
+    corresponds to x-axis
+    and k corresponds to the y-axis onto the axis that is ax. plot_ellipse then
+    applies a gmm ellipse onto the scatterplot
+    using the data from Y_(which is stored in data["clusters"]),
+    means, covariances.
+
+    Parameters
+    ----------
+    X : array-like, shape (n_samples, n_features)
+        Input data.
+    j : int
+        column index of feature of interest from X which will be the x-axis data.
+    k : int
+        column index of feature of interest from X which will be the y-axis data.
+    means : array-like, shape (n_components, n_features)
+        Estimated means from gmm
+    covariances : array-like, shape (with 'full') (n_components, n_features, n_features)
+        estimated covariances from gmm
+    ax : axis object
+        The location where plot_ellipse will plot
+    label_palette : dict, optional, default: dictionary using 'Set1'
+    cluster_palette : dict, optional, default: dictionary using 'Set1'
+    alpha : float, optional, default: 0.7
+        Opacity value of plotter markers between 0 and 1
+
+    References
+    ----------
+    .. [1] Weiss, R., Varoquaux, G.(2011). Scikit-learn: Machine Learning in PythonJournal
+        of Machine Learning Research, 12, 2825–2830.
+    """
```

Comment:
Apologies if not being clear, but I literally just wanted that specific link either in the comments in the code, or here. Just makes it more clear where we got that code.

Reply:
https://github.com/microsoft/graspologic/pull/541/commits/d5cb3cb733d0e9ae498c4eeb78eaca5ccce4d627 I think i fixed it. I am sorry i waas unsure what you meant.. hopefully this is right!